Makefile

```diff
@@ -22,7 +22,7 @@ image-load-cip-auditor-e2e:
 image-push-cip-auditor-e2e:
 	bazel run //test-e2e/cip-auditor:push-cip-auditor-test
 lint:
-	GO111MODULE=on golangci-lint --timeout=2s run
+	GO111MODULE=on golangci-lint run -v --timeout=5m 
```

Comment:
Can you move each flag to its own line with `\`? I don't like it when we have 1-line diffs and more than 1 thing is changed in it.